### PR TITLE
Update job-scheduler.md

### PR DESCRIPTION
**Pull request by schottj**
recent change to run slots per #shipped post

### CODE REVIEW

File: website/docs/docs/deploy/job-scheduler.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Familiarize yourself with these useful terms to help you understand how the job
 | Over-scheduled job | A situation when a cron-scheduled job's run duration becomes longer than the frequency of the job’s schedule, resulting in a job queue that will grow faster than the scheduler can process the job’s runs. |
 | Prep time | The time dbt Cloud takes to create a short-lived environment to execute the job commands in the user's cloud data platform. Prep time varies most significantly at the top of the hour when the dbt Cloud Scheduler experiences a lot of run traffic. |
 | Run | A single, unique execution of a dbt job. |
-| Run slot | Run slots control the number of jobs that can run concurrently. Each account has a fixed number of run slots, depending on the plan tier, that are shared across projects in the account. Each running job occupies a run slot for the duration of the run, so purchasing more run slots enables more jobs to execute in parallel. |
+| Run slot | Run slots control the number of jobs that can run concurrently. Developer and team plan accounts have a fixed number of run slots; whereas, enterprise customers have unlimited run slots. Each running job occupies a run slot for the duration of the run. |
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-scheduler.md]:
```suggestion
| Run slot | Run slots control the number of jobs that can run concurrently. Developer and Team plan accounts have a fixed number of run slots, and Enterprise users have [unlimited run slots](/docs/dbt-versions/release-notes/July-2023/faster-run#unlimited-job-concurrency-for-enterprise-accounts). Each running job occupies a run slot for the duration of the run. |
```

**Comment by mirnawong1** [comment]
thanks so much @schottj , added a link to the release note in case users wanted more context. 

quick question, we don't seem to mention dev and team plans having 2 run slots (not even on our [pricing page](https://www.getdbt.com/pricing/)), do you think we can add this like the below:

> Developer and Team plan accounts have two run slots, and...

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
hey @schottj , i think you opened pr https://github.com/dbt-labs/docs.getdbt.com/pull/3737/files, which makes the same changes as this pr. did you want to work on this pr first?

File: website/docs/docs/deploy/job-scheduler.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Familiarize yourself with these useful terms to help you understand how the job
...
 | Prep time | The time dbt Cloud takes to create a short-lived environment to execute the job commands in the user's cloud data platform. Prep time varies most significantly at the top of the hour when the dbt Cloud Scheduler experiences a lot of run traffic. |
 | Run | A single, unique execution of a dbt job. |
-| Run slot | Run slots control the number of jobs that can run concurrently. Each account has a fixed number of run slots, depending on the plan tier, that are shared across projects in the account. Each running job occupies a run slot for the duration of the run, so purchasing more run slots enables more jobs to execute in parallel. |
+| Run slot | Run slots control the number of jobs that can run concurrently. Developer and Team plan accounts have a fixed number of run slots, and Enterprise users have [unlimited run slots](/docs/dbt-versions/release-notes/July-2023/faster-run#unlimited-job-concurrency-for-enterprise-accounts). Each running job occupies a run slot for the duration of the run. |
```
Comment (mirnawong1) [inline website/docs/docs/deploy/job-scheduler.md]:
```suggestion
| Run slot | Run slots control the number of jobs that can run concurrently. Developer and Team plan accounts have a fixed number of run slots, and Enterprise users have [unlimited run slots](/docs/dbt-versions/release-notes/July-2023/faster-run#unlimited-job-concurrency-for-enterprise-accounts). Each running job occupies a run slot for the duration of the run. If you need more jobs to execute in parallel, consider the [Enterprise plan](https://www.getdbt.com/pricing/) |
```

**Comment by schottj** [comment]
I liked how you changed it:

| Run slot | Run slots control the number of jobs that can run concurrently. Developer and Team plan accounts have a fixed number of run slots, and Enterprise users have [unlimited run slots](/docs/dbt-versions/release-notes/July-2023/faster-run#unlimited-job-concurrency-for-enterprise-accounts). Each running job occupies a run slot for the duration of the run. |

I guess I forgot i had a pr open and tried to make the change 2x. Once you change it to the above, please feel free to merge!

**Review by mirnawong1 — APPROVED** [review]
🚢

**Pull request merged**